packages/shared/lib/services/connection.service.ts

```diff
@@ -492,6 +496,13 @@ class ConnectionService {
         return newMetadata;
     }
 
+    public async bulkUpdateMetada(connections: Connection[], metadata: Metadata): Promise<void> {
```

Comment:
can we not refactor `updateMetadata` to accept a array of connection instead of adding another public method?

Reply:
Sure: https://github.com/NangoHQ/nango/pull/2145/commits/294aa89bed2aa5984e8072081dbb096eb986040e